lib/vm/src/cache.rs

```diff
@@ -48,6 +59,13 @@ where
             modules,
             wasm_path,
             instances,
+            stats: Stats {
```

Comment:
rust tip:

if you add `#[derive(Default)]` above the `Stats` definition, then you get this via `Stats::Default()`

Reply:
Thanks! Done in de489a9dc286b736539d237c8a2648a8b86dc414